pkg/collector/corechecks/servicediscovery/model/model.go

```diff
@@ -0,0 +1,20 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package model contains types for service discovery.
+package model
+
+// Port represents an open port in the current host.
+type Port struct {
```

Comment:
Let's try to find better naming, as port struct that contains port field - feels like a naming issue